package.json

```diff
@@ -123,7 +123,7 @@
     "simple-get": "^3.0.3",
     "snazzy": "^8.0.0",
     "split2": "^3.1.0",
-    "standard": "^12.0.1",
+    "standard": "^13.0.1",
```

Comment:
```suggestion
    "standard": "^13.0.2",
```

Reply:
There is no need to do this, we are not using a lockfile and npm will take care of installing the latest compatible version :)